### DISCUSSION TITLE
New v0.2.2 version

### DISCUSSION THREAD

**Discussion by 4rtzel** [4rtzel/poe-archnemesis-scanner#57] [Announcements]
* Fix scanner window calculation for 1920x1080 resolutions.
* Add debug command line options to help with troubleshooting:
  * --show-capture-image -- display the image that was captured by scanner window.
  * --scanner-window-{x,y,width,height} -- sets scanner window position and size.
* Add output message for the number of items found.

<hr /><em>This discussion was created from the release <a href='https://github.com/4rtzel/poe-archnemesis-scanner/releases/tag/'>New v0.2.2 version</a>.</em>

**Comment by patjbarker** [comment]
For 5120x1440, what would you set the image scale to so the algorithm works properly?

**Reply by 4rtzel** [reply]
Try the default settings first and see if it works.

**Reply by patjbarker** [reply]
Yeah, it works. I just wanted to inquire as to whether due to the res there wasn't a better value. Thanks.

**Reply by 4rtzel** [reply]
The tool tries to calculate the best value based on your resolution. The calculations might not yield the perfect values but it seems to work quite well.
